Makefile

```diff
@@ -29,13 +28,13 @@ lint: ## Lint the code
 	poetry run ruff format --check && \
 	poetry run ruff check && \
 	poetry run mypy .
-	npx prettier --check .
+	docker compose run --rm -v $(shell pwd):/work app npx prettier --check ./*.md ./docs ./.github/workflows/* ./hushline
```

Comment:
when running with `docker compose` you don't have to re-specify the volumes as they already get pulled in. The `workdir` will be `/app` and the repo root will be mounted at `/app`, so this should do what we need without a second mount, no?